src/index.html

```diff
@@ -183,6 +183,121 @@ <h2 class="intro__heading">Dream. Develop. Deploy!</h2>
+                <div class="content">
+                  <p>We’ll be sending buses to Toronto, Waterloo, and Boston. You can take the Via Rail to Montreal Central Station, a short walk from McGill campus. You can also take the Montreal metro to Peel station, our closest stop.</p>
+                </div>
+              </div>
+            </div>
+            <br>
+            <div class="background">
+              <div class="accordion-item">
+                <a>Are travel reimbursements provided?</a>
+                <div class="content">
+                  <p>800 on Moh's Scale</p>
+                </div>
+              </div>
+            </div>
+            <br>
+            <div class="background">
+              <div class="accordion-item">
+                <a>Is there a standby line on the day of the event?</a>
+                <div class="content">
+                  <p>If we have extra spots available, we will post on our Facebook page in the late morning (around 10:30-11AM). While we wish we could have each and every applicant at McHacks, we were unfortunately unable to book Hilbert’s Infinite Hotel this year.</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="right-container">
+          <div class="accordion">
+            <div class="background">
+              <div class="accordion-item">
+                <a>I don’t have a team. Can I still participate?</a>
+                <div class="content">
+                  <p>Of course, it’s not necessary to have a team prior to McHacks! We’ll have a fun team-forming session where you can find an awesome group of people to work with. Teams are welcome to have up to four people.</p>
+                </div>
+              </div>
+            </div>
+            <br>
+            <div class="background">
+              <div class="accordion-item">
+                <a>How much is this going to cost me?</a>
+                <div class="content">
+                  <p>McHacks is completely free! We’ll be providing food, drink, wi-fi, and lots of awesome swag. :)</p>
+                </div>
+              </div>
+            </div>
+            <br>
+            <div class="background">
+              <div class="accordion-item">
+                <a>Can I keep working on a past project?</a>
+                <div class="content">
+                  <p>No, participants should not be working on their projects before McHacks nor be developing pre-existing projects during the hackathon.</p>
+                </div>
+              </div>
+            </div>
+            <br>
+            <div class="background">
+              <div class="accordion-item">
+                <a>Is parking available?</a>
+                <div class="content">
+                  <p>Parking is available at the McIntyre Garage (3655 Promenade Sir-William-Osler, Montreal, QC H3G 0B1). Rates are a max of $10.00 between 6:00 am and midnight on weekends.</p>
```

Comment:
This is not an accurate description of the parking situation for St James.

Reply:
What would be an accurate description of the parking situation?